lang/java/compiler/src/test/java/org/apache/avro/compiler/specific/TestSpecificCompiler.java

```diff
@@ -300,8 +305,8 @@ public void testSettingOutputCharacterEncoding() throws Exception {
     is.close(); // close input stream otherwise delete might fail
     if (!this.outputFile.delete()) {
       throw new IllegalStateException("unable to delete " + this.outputFile); // delete otherwise compiler might not
-                                                                              // overwrite because src timestamp hasn't
-                                                                              // changed.
+      // overwrite because src timestamp hasn't
```

Comment:
Please restore the formatting or move the whole comment on the previous line

Reply:
Not sure how that occurred, but reverted